clumper/clump.py

```diff
@@ -62,6 +62,56 @@ def read_json(cls, path):
             return Clumper(data)
         return Clumper(json.loads(pathlib.Path(path).read_text()))
 
+    @classmethod
+    def read_jsonl(cls, path: str, lines=None):
+        """
+        Reads in a jsonl file. You can also specify how many lines you want to read in.
+
+        Usage:
+
+        ```python
+        from clumper import Clumper
+
+        clump = Clumper.read_jsonl("tests/cards.jsonl")
+        assert len(clump) == 5
+
+
+        clump = Clumper.read_jsonl("https://calmcode.io/datasets/pokemon.jsonl")
+        assert len(clump) == 800
+
+        """
+
+        assert path.lower().endswith(
```

Comment:
I think in the future I might change this to a `ValueError` but that's really a detail.